perception/sub8_perception/CMakeLists.txt

```diff
@@ -133,6 +133,10 @@ install(DIRECTORY include/${PROJECT_NAME}/
 #     "src/sub8_perception/*.cc"
 # )
 
+file(GLOB START_GATE_SOURCES
+      "src/sub8_perception/start_gate.cc"
```

Comment:
*pushes back glasses* umm actually the [official ros style guide](http://wiki.ros.org/CppStyleGuide#Files) states that c++ header files should have the extension **.h**